inc/html.class.php

```diff
@@ -4695,13 +4693,11 @@ static function jsAjaxDropdown($name, $field_id, $url, $params = []) {
       unset($params['value']);
       unset($params['valuename']);
 
-      if (!empty($params['specific_tags'])) {
```

Comment:
Normalization of parameters handling ensures there is always a "specific_tags" key; then the if statement becomes useless.